frontends/open-discussions/src/components/ImageUploaderForm.js

```diff
@@ -6,7 +6,7 @@ import Dropzone from "react-dropzone"
 import CropperWrapper from "./CropperWrapper"
 import Dialog from "./Dialog"
 
-import { validationMessage } from "../lib/validation"
+import { ValidationError } from "ol-forms"
```

Comment:
All `validationMessage` usages could/should be replaced with ValidationError. But enzyme is a little brittle about replacing components like this, this breaks a few tests. I'd like to keep that replacement for a future PR since this is already big.